doc/dev/tests.md

```diff
@@ -5,24 +5,42 @@ title: Tests
 
 ## Overview
 
-Npgsql comes with an extensive test suite to make sure no regressions occur. All tests are run on our build server on all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
+Npgsql has an extensive test suite to guard against regressions. The test suite is run on the official build server for all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
```

Comment:
Sorry for not mentioning this earlier, but we don't actually test on mono on Appveyor or Travis - only on .NET Framework and .NET Core - better to say that instead.

Reply:
How's that look?